src/ScClient.ts

```diff
@@ -122,21 +128,31 @@ export class ScClient {
     sendData();
   }
 
+  private formSendAnswer(resolve: (arg: any) => void, reject: (arg: any) => void, success: any, errors?: any) {
```

Comment:
seems like we know some types.

```suggestion
  private formSendAnswer(resolve: (arg: any) => void, reject: (arg: Error[]) => void, success: any, errors?: Error[]) {
```